fastlane/Fastfile

```diff
@@ -319,11 +321,37 @@ ENV["HAS_ALPHA_VERSION"]="true"
     {library: "Image Editor", strings_path: "./libs/image-editor/ImageEditor/src/main/res/values/strings.xml", exclusions: []}
   ]
 
+  private_lane :commit_strings do | options |
+    if (options[:auto_commit]) then
+       sh("cd .. && git add #{main_strings_path}")
+       sh("git commit -m 'Update strings for translation'")
+       sh("git push") 
```

Comment:
Can we do `git push origin HEAD` here? Just to make sure the local git config doesn't interfere?

Reply:
Ah! Thank you for catching this @maxme! 
t seems it's hard for me to get rid on this old habit! I'm sorry for having missed it again!
This is now updated!